source/custom_templates/_tx_api_page.txt

```diff
@@ -1,8 +1,9 @@
 .. include:: _:page:.tmp
 
-.. include:: common_response_params.rst
+.. include:: _:org_type:_param_table.rst
 
-.. include:: optional_fields.txt
+The following fields are optional based on your account type:
 
+.. include:: _:org_type:_optional_param_table.rst
 
```

Comment:
I think so, would be nice if we had an overall header for the section, and then each table could be annotated in some way:

----------

## Optional Parameters

#### Signal Parameters
[table]

#### Conversion Reporting Parameters
[table about call center integration fields]

#### Additional Features
[table with the rest]

Please collaborate with Nathan and get his input on the headers / grouping, but I think something like the above should work.